### PR TITLE
Improve algo

### PR DESCRIPTION
**Pull request by amitojsingh**
There was problems in levelsfooter on end session the current progress and current level wasn't stored properly. so In this PR I fixed this.

### CODE REVIEW

**Comment by MySmilingTurban** [comment]
Please add description and fix deployment issues
@amitojsingh

**Comment by MySmilingTurban** [comment]
Can we also add Back button in this or in another PR?

**Comment by amitojsingh** [comment]
done @MySmilingTurban can you check now?

File: src/pages/dashboard/hooks/useGamePlay1.ts [hunk 1]
```diff
@@ -15,14 +15,20 @@ const useGamePlay = (user: User, toggleLoading: (value: boolean) => void, resetG
   const dispatch = useAppDispatch();
   const getRandomQuestions = useQuestions(user);
   const getNewQuestions = useNew();
-  const inProgressWords:WordShabadavaliDB[] = [];
-  
+  const inProgressWords: WordShabadavaliDB[] = [];
+
   const gamePlay = async () => {
-    const progress: GameScreen[] | null = fetchProgress(user);
+    const usr = await getUserData(user.uid);
```
Comment (MySmilingTurban) [inline src/pages/dashboard/hooks/useGamePlay1.ts:21]: rename it to userData?

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
One minor change

File: src/pages/dashboard/hooks/useGamePlay1.ts (hunk 1 above)
Comment (amitojsingh) [reply src/pages/dashboard/hooks/useGamePlay1.ts]: fixed

**Pull request merged**